### PR TITLE
Feature/configurable rsync settings

### PR DESCRIPTION
**Pull request by mstruebing**

### CODE REVIEW

File: Classes/Sitegeist/MagicWand/Command/CloneCommandController.php [hunk 1]
```diff
@@ -319,7 +319,7 @@ protected function checkConfiguration($remotePersistenceConfiguration)
     protected function renderRsyncParams()
     {
         foreach ($this->rsyncParams as $index => $param) {
-            $index == 0 ? $rsyncParams = $param : $rsyncParams .= ' ' . $param;
+            $index === 0 ? $rsyncParams = $param : $rsyncParams .= ' ' . $param;
```
Comment (grebaldi) [inline Classes/Sitegeist/MagicWand/Command/CloneCommandController.php]: Wouldn't `implode(' ', $this->rsyncParam)` do the job?

**Comment by mficzel** [comment]
I would like to configure it in general and per preset if needed. 

Maybe flowComnand and ssh / rsync Settings should go into an option setting that exist globally and (optional) per preset.

**Comment by grebaldi** [comment]
How shall we proceed with this one? @mficzel @mstruebing

**Comment by mstruebing** [comment]
I think we can close it because it was my idea to use rsync with the `--progress` flag, which doesn't have any usable effect because `shell_exec` (https://github.com/sitegeist/Sitegeist.MagicWand/blob/master/Classes/Sitegeist/MagicWand/Command/AbstractCommandController.php#L58) isn't giving the output of the command until it is finshed.

**Pull request closed without merging**